target/linux/mvebu/image/Makefile

```diff
@@ -30,6 +30,10 @@ define Build/boot-scr
 	mkimage -A arm -O linux -T script -C none -a 0 -e 0 -d $(DEVICE_NAME).bootscript $@-boot.scr
 endef
 
+define Build/boot-scr-dummy
+	echo "# Boot script not used for this target" > $@-boot.scr
+endef
```

Comment:
Because of this line:
https://github.com/openwrt/openwrt/blob/11d70f0600921702af63c10a9482029b072dbf0b/target/linux/mvebu/image/Makefile#L38
this define is unnecessary. It is allowed to fail if there is no boot.scr present.